### ISSUE TITLE
Bug: Creation of duplicate organizations throws error

### ISSUE THREAD

**Issue by meenakshi-dhanani** [anitab-org/bridge-in-tech-backend#110]
**Describe the bug**
Creation of a duplicate organization throws error and a 500.

**To Reproduce**

As a user, when I try to create an organization it creates successfully, when I try creating another organization with the same payload I see the following error:

<img width="1367" alt="Screen Shot 2020-08-16 at 9 23 23 PM" src="https://user-images.githubusercontent.com/14279061/90338505-ba1bd300-e007-11ea-9e20-5e944567f89d.png">

**Expected behavior**
409 for duplicate resource exists should be returned along with appropriate error message

**Comment by v1shwa** [comment]
Hi @meenakshi-dhanani, I can work on fixing this, could you please assign this to me.

**Comment by mtreacy002** [comment]
Sure, @v1shwa . I'll assign you to thiis.

**Comment by v1shwa** [comment]
Hi @meenakshi-dhanani Could you please provide more details about the bug. I tried to replicate the error with the provided instructions but couldn't find any issue.

When I create a new org, I get the response: `Organization was created successfully.` and when I re-submit the same payload again, it says: `Organization was updated successfully.` which I believe is the expected behavior.

It'd be really helpful if you can share the exact procedure to reproduce the bug including the payloads you've used. Thanks! cc: @mtreacy002

**Comment by mtreacy002** [comment]
@v1shwa, can you try to create the same organization name but with different user? Perhaps this will trigger the bug.

**Comment by mtreacy002** [comment]
@v1shwa can you please give me an update on this issue?

**Comment by mtreacy002** [comment]
This issue is now available for anyone who's interested.

**Comment by ashwinijha6** [comment]
@mtreacy002 I would like to work on this issue could u pls assign this to me

**Comment by mtreacy002** [comment]
suree, @ashwinijha6 . I'll assign you to this. Thanks for picking it up.

**Comment by NenadPantelic** [comment]
Hi @mtreacy002 @ashwinijha6. @ashwinijha6  can you tell me if you work on this or it is available? :slightly_smiling_face:

**Comment by ashwinijha6** [comment]
Yeah I'm working on it, will require some more time, actually was getting an internal server error.
If I will not be able to find the bug will let u know.

**Comment by NenadPantelic** [comment]
@ashwinijha6 Deal :+1:

**Comment by mtreacy002** [comment]
@ashwinijha6 , any update here?

**Comment by ashwinijha6** [comment]
> @ashwinijha6 , any update here?

I'm really sorry @mtreacy002 I'm a bit preoccupied at the moment. If someone wants to take up the issue can go for it.

**Comment by NenadPantelic** [comment]
@mtreacy002  I can take this one if @ashwinijha6 is occupied :slightly_smiling_face:

**Comment by mtreacy002** [comment]
sure, @NenadPantelic . will assign you to this. @ashwinijha6 , thanks for letting us know 😉

**Comment by NenadPantelic** [comment]
Sorry for being inactive in the last three days, I had some private things to do. I'll start working on this very soon.

**Comment by mtreacy002** [comment]
@NenadPantelic , any updates on this?

**Comment by NenadPantelic** [comment]
Hope to open the PR today or tomorrow.
